### ISSUE TITLE
Cloning instructions for different steps

### ISSUE THREAD

**Issue by goldylucks** [verekia/js-stack-from-scratch#183]
### Type of issue: (feature suggestion, bug?)
Docs

### Chapter:
All

I'm now cloning [step 4](https://github.com/verekia/js-stack-walkthrough/tree/master/04-webpack-react-hmr) to build a small app. Would be beneficial to have a README file with instructions how to clone that specific part (and all the others)

I can prepare a PR :)

**Comment by verekia** [comment]
Good call. I've wanted to clone a specific step myself in the past too. Hmm. Maybe I should add one zip per chapter in the releases.

**Comment by goldylucks** [comment]
I haven't done it before, tho I'm pretty sure we can make a small bash script that will download the repo at a certain branch/commit and initialise a new repo

Tho probably a zip version would be good enough :)
